couchbase/admin/collection/v1/collection.proto

```diff
@@ -56,10 +59,18 @@ message CreateCollectionRequest {
   string scope_name = 2;
   string collection_name = 3;
   optional uint32 max_expiry_secs = 4;
+  optional bool history_retention_enabled = 5;
 }
 
 message CreateCollectionResponse {}
 
+message UpdateCollectionRequest {
```

Comment:
I think this will need the bucket name as well, since in `gocbcoreps` we need to do:
```
func (c *routingImpl_CollectionV1) UpdateCollection(ctx context.Context, in *admin_collection_v1.UpdateCollectionRequest, opts ...grpc.CallOption) (*admin_collection_v1.UpdateCollectionResponse, error) {
	return c.client.fetchConnForBucket(in.BucketName).CollectionV1().UpdateCollection(ctx, in, opts...)
}
```

Reply:
Also I think we'll need the scope and collection name, since we need to pass them from STG to gocbccorex for the URL

Reply:
We'll need all 3 so that anything can identify which collection it is, oops.